src/EspMQTTClient.h

```diff
@@ -157,6 +165,8 @@ class EspMQTTClient
   inline const char* getMqttClientName() { return _mqttClientName; };
   inline const char* getMqttServerIp() { return _mqttServerIp; };
   inline const short getMqttServerPort() { return _mqttServerPort; };
+  inline void disconnectMqtt() { _mqttClient.disconnect(); };
```

Comment:
Not sure if this is really needed? looks like you tested with it in the example code but stopped using it. Maybe also get rid of this addition then?

Reply:
Yes I used it because long time ago OTA while working with other connected TCP was difficult to go until the end without errors, but looks like now it correctly handed by the framework. But does not hurt to leave it, if some have issues during OTA it's the first thing to do, close all other network connectivity when starting OTA